src/main/resources/liquibase/crop_changelog/19_4_0.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cvterm" columnName="is_system"/>
+			</not>
+		</preConditions>
+		<comment>Adding is_system column</comment>
+		<addColumn tableName="cvterm">
+			<column name="is_system" type="TINYINT(1) UNSIGNED ZEROFILL" defaultValue="0" afterColumn="is_relationshiptype">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+    </changeSet>
+
+	<changeSet author="cuenyad" id="v19.4.0-2">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="cvterm" columnName="is_system"/>
+		</preConditions>
+		<comment>Mark system variables</comment>
+        <sql>
+			update cvterm cv INNER JOIN cvterm cv1 ON cv.cvterm_id = cv1.cvterm_id  set cv.is_system = 1
```

Comment:
why an inner join is needed? in both cases it does not seem to be necessary! @cuenyad